clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -139,6 +139,7 @@ void visualstudio::Linker::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back("-defaultlib:sycld.lib");
     else
       CmdArgs.push_back("-defaultlib:sycl.lib");
+    CmdArgs.push_back("-defaultlib:sycl-devicelib-host.lib");
```

Comment:
Hi, @mdtoguchi 
The library "sycl-devicelib-host.lib" contains implementation for imf device library functions for SYCL host device. This static library will only be used when users are using sycl compiler.

Thanks very much.